test/metrics/NumberMethods/1.java

```diff
@@ -0,0 +1,34 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Hospital {
+
+	public static List<Instance> readDataGet(String file) throws FileNotFoundException {
+		List<Instance> dataset = new ArrayList<Instance>();
+		Scanner scanner = null;
+		try {
+			scanner = new Scanner(new File(file));
+			while(scanner.hasNextLine()) { // +1
```

Comment:
Here and in all others `*.java` files please put such comments to the lines which affects current metric.
Also do not just copy files for unit test. Write it your self to make it minimal as possible. It will be easier for debugging.

Reply:
ok